charts/sonarqube/templates/deployment.yaml

```diff
@@ -101,7 +101,7 @@ spec:
       {{- end }}
       {{- if .Values.plugins.install }}
         - name: install-plugins
-          image: {{ default "alpine:3.10.3" .Values.plugins.initContainerImage }}
+          image: {{ default "rjkernick/alpine-wget:latest" .Values.plugins.initContainerImage }}
```

Comment:
Thanks for the fix, but is this really required? Why not just use curl to grab the plugins, as suggested in `helm/charts#18805`? This means you don't have to maintain this one-off image, and we can keep the trusted official source as the default.

Reply:
@usmonster you are absolutely right. It is going to be tricky:  alpine:3.10.3 -> `/bin/sh: curl: not found`

Reply:
Ah! I wasn't aware. 😞  Thanks for the quick response!

Reply:
Ah! I wasn't aware. 😞  Thanks for the quick response!